ddtrace/contrib/trace_utils.py

```diff
@@ -103,3 +104,14 @@ def ext_service(pin, config, default=None):
 
     # A default is required since it's an external service.
     return default
+
+
+def set_http_meta(config, span, method=None, url=None, status_code=None):
```

Comment:
Just a personal preference, but I'd prefer that the span be the first argument of the helper